brd-android/app/src/main/java/com/breadwallet/ui/navigation/NavigationTarget.kt

```diff
@@ -80,7 +80,8 @@ sealed class NavigationTarget : INavigationTarget {
 
     data class FabriikToast(
         val type: Type,
-        val message: String
+        val message: String? = null,
+        val messageRes: Int? = null
```

Comment:
could you add here StringRes annotation? to make sure the int comes from String resources?